lib/nexmo/oas/renderer/views/open_api/_model.erb

```diff
@@ -2,7 +2,7 @@
   <div class="Vlt-col Vlt-col--2of3 Nxd-api__docs">
     <div>
       <h2 id="<%= group['name'].parameterize %>" class='Vlt-grey-dark'><%= group['name'] %></h2>
-      <p class: "Vlt-grey-darker"><%= group['description'].render_markdown %></p>
+      <p class: "Vlt-grey-darker"><%= group['description']&.render_markdown %></p>
```

Comment:
Is this because sometimes `group['description']` was/is Nil?

Reply:
yes :D

Reply:
@benhayehudi  if you run it locally with:

```
OAS_PATH=../api-specification/definitions bundle exec shotgun lib/nexmo/oas/renderer/config.ru
```

and visit http://127.0.0.1:9393/vonage-business-cloud/vgis

you'll get the error

Reply:
Thanks, that's what I thought, just wanted to make sure 👍